lib/Dialect/FIRRTL/Transforms/LowerClasses.cpp

```diff
@@ -1238,16 +1194,6 @@ void LowerClassesPass::lowerClass(om::ClassOp classOp, FModuleLike moduleLike,
   // If the module-like is a Class, it will be completely erased later.
   // Otherwise, erase just the property ports and ops.
   if (!isa<firrtl::ClassLike>(moduleLike.getOperation())) {
-    // Erase ops in use before def order, thanks to FIRRTL's SSA regions.
-    for (auto &op :
```

Comment:
I'm not 100% if this is safe to delete, see:
https://github.com/llvm/circt/pull/7823/files#r1844569707

cc @mikeurbach @uenoku

Reply:
Seems like it is safe to remove